src/components/views/JobInfo.js

```diff
@@ -0,0 +1,205 @@
+      </Pane>
+    );
+  }
+
+  getSectionProps = (id) => {
+    return {
+      id,
+      onToggle: this.handleSectionToggle,
+      open: this.state.sections[id],
+    };
+  }
+
+  handleSectionToggle = ({ id }) => {
+    this.setState((prevState) => ({
+      sections: {
+        ...prevState.sections,
+        [id]: !prevState.sections[id],
+      }
+    }));
+  }
+
+  handleAllSectionsToggle = (sections) => {
+    this.setState({ sections });
+  }
+
+  render() {
+    const { data: { job }, isLoading } = this.props;
+
+    if (isLoading) return this.renderLoadingPane();
+    const isJobQueued = get(job, 'status.value') === 'queued';
+    const isJobInProgress = get(job, 'status.value') === 'in_progress';
+
+    return (
+      <Pane
+        defaultWidth="45%"
+        dismissible
+        id="pane-view-job"
+        onClose={this.props.onClose}
+        paneTitle={job.name}
+      >
+        <TitleManager record={job.name}>
+          <div>
+            <Row>
+              <Col xs={9}>
+                <KeyValue label={<FormattedMessage id="ui-local-kb-admin.prop.jobName" />}>
+                  <div data-test-job-name>
+                    <strong>{job.name}</strong>
+                  </div>
+                </KeyValue>
+              </Col>
+              {
+                isJobInProgress ? (
+                  <Col xs={2} xsOffset={1}>
+                    <Button
+                      buttonStyle="danger"
+                      id="clickable-delete-job"
+                      onClick={() => {}}
+                    >
+                      <FormattedMessage id="ui-local-kb-admin.job.delete" />
+                    </Button>
+                  </Col>) : null
+              }
+            </Row>
+            <Row>
+              <Col xs={4}>
+                <KeyValue label={<FormattedMessage id="ui-local-kb-admin.prop.runningStatus" />}>
+                  <div data-test-job-status>
+                    {get(job, 'status.label', '-')}
+                  </div>
+                </KeyValue>
+              </Col>
+              <Col xs={4}>
+                {
+                  !isJobQueued && (
+                    <KeyValue label={<FormattedMessage id="ui-local-kb-admin.prop.outcome" />}>
+                      <div data-test-job-result>
+                        {get(job, 'result.label', '-')}
+                      </div>
+                    </KeyValue>
+                  )
+                }
+              </Col>
+              <Col xs={4}>
+                {
+                  !isJobQueued && (
+                    <KeyValue label={<FormattedMessage id="ui-local-kb-admin.prop.errors" />}>
+                      <div data-test-job-errors>
+                        {job.errorLog ? job.errorLog.length : '0'}
+                      </div>
+                    </KeyValue>
+                  )
+                }
+              </Col>
+            </Row>
+            <Row>
+              <Col xs={!isJobQueued ? 4 : 0}>
```

Comment:
This field (and similarly for the `ended` field) could be cleaned up logically by moving the `!isJobQueued` check to outside the `Col` rather than checking if it's queued twice. Eg,
```
{ !isJobQueued && (
  <Col xs={4}>
    <KeyValue>...</KeyValue>
  </Col>
) }```